components/inbound-endpoints/org.wso2.carbon.inbound.endpoint.persistence/src/main/java/org/wso2/carbon/inbound/endpoint/persistence/PersistenceUtils.java

```diff
@@ -294,5 +305,23 @@ private static InboundProcessorParams deserializeInboundParameters(OMElement end
         inboundParams.setProperties(props);
         return inboundParams;
     }
-
+    public static int getPortOffset(Properties properties) {
```

Comment:
Need a method comment here. Better to explain the logic since this does not simply return the port offset.